dags/gmail_data_import_pipeline.py

```diff
@@ -116,129 +115,94 @@ def get_gmail_service():
     )
 
 
-def gmail_label_data_to_temp_table_etl(**kwargs):
-    data_config = data_config_from_xcom(kwargs)
-    user_id = get_gmail_user_id()
-    project_name = data_config.project_name
-    dataset_name = data_config.dataset_name
-    table_name = data_config.temp_table_name_labels
+def get_gmail_df_to_load_bq_table(
```

Comment:
Maybe we could try to find a different name. Just because `get` usually implies it is returning something and also wouldn't usually have a side effect... maybe turning it around to something like: `load_bq_table_from_df`?